src/bp-core/classes/class-bp-admin-types.php

```diff
@@ -86,7 +84,7 @@ public static function register_types_admin() {
 			$bp->core->types_admin = new self;
 		}
 
-		return $bp->core->types_admin;
+		$bp->core->types_admin;
```

Comment:
This change is making me nervous, I'd like to be sure it doesn't break the Types Admin first by testing it.

Reply:
tested this and didn't see any issues or warnings.

Reply:
Just tested this. I saw no side effects. It's could to go.